plugin-tester-java/src/main/protobuf/helloworld.proto

```diff
@@ -38,4 +40,5 @@ message HelloRequest {
 // The response message containing the greetings
 message HelloReply {
     string message = 1;
+    google.protobuf.Timestamp timestamp = 2;
```

Comment:
This test project now fails for the non-maven builds (sbt and gradle):
```
helloworld.proto:3:1: Import "google/protobuf/timestamp.proto" was not found or had errors.
helloworld.proto:43:5: "google.protobuf.Timestamp" is not defined.
```

Reply:
Could you, please, rename the issue. I will go for doing the same for the gradle and sbt as well